modules/exploits/linux/http/kaltura_unserialize_cookie_rce.rb

```diff
@@ -85,12 +89,29 @@ def check
       Exploit::CheckCode::Safe
     elsif res && res.body.include?(r)
       Exploit::CheckCode::Vulnerable
+    elsif not self.check_entryid()
```

Comment:
Do you really need to use `self` and `()` in here ? I sense python developer in here :-)

Reply:
Shouldn't we check validity of entry_id way before ? I would call `check_entryid` at the very beginning of `check` and `exploit` functions.

Reply:
I want to keep the number of requests as low as possible, so `check` should only test for an invalid id if the request before failed and `exploit` should not check for a valid id every time a call is made. I'll remove the `self.` and `()`.